### PR TITLE
Bump up kolla-ansible version and whitelist rabbitmq queues

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

**Review by dougszumski — APPROVED** [review]
Thanks @brtknr 

For context the Kolla-Ansible bump is to pull in a fix for host specific config overrides in the Monasca role.

File: etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml [hunk 1]
```diff
@@ -13,8 +13,25 @@ instances:
   name: RabbitMQ
   nodes:
   - "rabbit@{{ inventory_hostname }}"
-  queues:
-  - conductor
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml:17]: No more conductor?

Comment (brtkwr) [reply etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml:17]: Do we need it @dougszumski ?

Comment (dougszumski) [reply etc/kayobe/kolla/config/monasca/sv-b16-u23/agent_plugins/rabbitmq.yaml:17]: I think it came from the default plugin config. For the Rabbit dashboard that we have, we focus on unacknowledged messages and message rates, which include the conductor (I assume it is Nova conductor). This PR matches the config we have for a working dashboard in another deployment. I think it's good to go.

**Review by dougszumski — APPROVED** [review]

**Review by markgoddard — APPROVED** [review]
If @dougszumski's happy then I'm happy.

**Pull request merged**